libs/features/personalization/personalization.js

```diff
@@ -291,13 +296,44 @@ function getSection(rootEl, idx) {
     : rootEl.querySelector(`:scope > div:nth-child(${idx})`);
 }
 
-function registerCustomAction(cmd, manifestId) {
-  const { action, selector, target } = cmd;
+function registerInBlockActions(cmd, manifestId) {
+  const { action, target, selector } = cmd;
+  const command = { action, target, manifestId };
+
+  const blockAndSelector = selector.substring(CUSTOM_SELECTOR_PREFIX.length).trim().split(/\s+/);
```

Comment:
`CUSTOM_SELECTOR_PREFIX` rename to `INBLOCK_SELECTOR_PREFIX`?

Reply:
Good call.  Also realized there's no reason to export it.